src/main/java/org/openelisglobal/common/services/NotificationService.java

```diff
@@ -0,0 +1,22 @@
+package org.openelisglobal.common.services;
+
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+import org.openelisglobal.login.valueholder.LoginUser;
+import org.openelisglobal.spring.util.SpringContext;
+import java.sql.Date;
+import org.openelisglobal.login.service.LoginUserService;
+
+@Service
+@Scope("prototype")
+@DependsOn({ "springContext" })
+public class NotificationService {
+
+    private  LoginUserService loginService = SpringContext.getBean(LoginUserService.class);
```

Comment:
i think its better to just Autowire the LoginUserService here since this class is a bean .
its a better Option to let spring handle beans than mannually accessing them